six/modules/c++/six.sidd/tests/test_geotiff.cpp

```diff
@@ -102,33 +102,25 @@ void write(const sys::Int16_T* data)
     writer.save(reinterpret_cast<const six::UByte*>(data), OUTPUT_NAME);
 }
 
-void read(const std::string& filename, sys::Int16_T* data)
+void read(const std::string& filename, mem::ScopedArray<sys::Int16_T>& data)
 {
-    six::sidd::GeoTIFFReadControl reader;
+    six::NITFReadControl reader;
```

Comment:
Does this test program really get run?  This can't be right... it should still be a `GeoTIFFReadControl` right??